modules/microshift-lvms-deployment.adoc

```diff
@@ -6,7 +6,11 @@
 [id="lvms-deployment_{context}"]
 = LVMS Deployment
 
-LVMS is automatically deployed on to the cluster in the `openshift-storage` namespace after {product-title} boots.
-//Q: is this correct, or should it be `microshift-namespace`?
+LVMS is automatically deployed on to the cluster in the `openshift-storage` namespace after {product-title} starts.
 
 LVMS uses `StorageCapacity` tracking to ensure that pods with an LVMS PVC are not scheduled if the requested storage is greater than the free storage of the volume group. For more information about `StorageCapacity` tracking, read link:https://kubernetes.io/docs/concepts/storage/storage-capacity/[Storage Capacity].
+
+[id="microshift-lvms-setting-path_{context}"]
+== Setting the LVMS path
+
+The `lvmd.yaml` file for the LMVS must be written to the same directory as the {product-title} `config.yaml` file. When {product-title} runs, the `/etc/microshift/lvmd.yaml` directory and file are accessed.
```

Comment:
This could be clearer. I think the important detail is that the LVMD config file goes in `/etc/microshift/lvmd.yaml`. In _older_ versions, it could live next to the MicroShift config file in one of several places, but we only look in `/etc/microshift` for it now, so the first sentence doesn't add anything.

```suggestion
When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided.
```

Reply:
Done. Do we need to give folks a command to create the file?